test/blackbox-tests/test-cases/variants/run.t

```diff
@@ -21,39 +21,89 @@ virtual libraries may not implement their virtual modules
   [1]
 
   $ cd module-fields && ocaml test.ml
-  impl: true. modules_without_implementation: true. virtual_modules: true
+  impl: true. modules_without_implementation: true. virtual_modules: true. private_modules: true
```

Comment:
Could you make `private_modules` be the parameter that changes last? Just to minimize the diff

Reply:
Sure. I fixed the order.